lib/msal-common/src/account/JwtToken.ts

```diff
@@ -11,39 +11,39 @@ import { ICrypto } from "../crypto/ICrypto";
 /**
  * Id Token representation class. Parses id token string and generates claims object.
  */
-export class IdToken {
+export class JwtToken {
```

Comment:
This is redundant, and I don't think the name should include JWT, as JWT is an implementation detail.

Reply:
JWTs are a specific format of token, and this class is built to that specific implementation. So I'm not sure I see the issue...?

Reply:
You're right, but generally speaking, I personally try to avoid leaking implementation details, including in names.

Also, JwtToken is redundant. Maybe `AuthToken`?

Reply:
done.